uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-infinispan/src/main/java/org/uberfire/ext/metadata/backend/infinispan/proto/KObjectMarshaller.java

```diff
@@ -105,6 +110,13 @@ public KObjectImpl readFrom(ProtoStreamReader protoStreamReader) throws IOExcept
                                !StringUtils.isEmpty(fullText));
     }
 
+    private String getAndRemove(final List<KProperty<?>> properties,
+                                final String key) {
+        KProperty<?> value = properties.stream().filter(kProperty -> kProperty.getName().equals(key)).findFirst().get();
```

Comment:
@adrielparedes as discussed you said that all the properties are mandatory, so it is safe to use `Optinal.get()`.
But it might still be better to use `optional.orElseThrow(() -> new IllegalStateException("Missing mandatory property " + key));` to get better error message in case it fails for some unexpected reason and to avoid IntelliJ's warning "get without isPresent check"